demo/doctest/hw1.py

```diff
@@ -27,3 +27,11 @@ def double(x):
     """
     return x # Incorrect
 
+def forever():
```

Comment:
This PR only affects `sqlite.py`, but this demo seems to imply it affects doctests as well. Is that the case?